src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
```

Comment:
What do we use this for? Is it for Jest or Enzyme?

Reply:
https://create-react-app.dev/docs/measuring-performance/

Reply:
It was scaffolded with the react app, it's for performance

Reply:
Okey, maybe I / we deleted it in other projects? :)